.eslintrc

```diff
@@ -19,6 +18,8 @@
     "import/no-extraneous-dependencies": [
       "error",
       { "devDependencies": [".storybook/**", "stories/**", "**/*.stories.jsx"] }
-    ]
+    ],
+    "react/jsx-props-no-spreading": [0],
+    "max-len": [1, 120]
   }
 }
```

Comment:
are we making it a requirement that all files have a newline at the bottom?

Reply:
Yes! my editor's linter was crashed yesterday and as a result we missed a bunch of these. I will fix them all, thanks!